### ISSUE TITLE
Adding images to an existing scene

### ISSUE THREAD

**Issue by vishaal-sridhar** [openMVG/openMVG#1321]
So I have a structured already constructed using openMVG and openMVS. I now have 2 more pictures I want to add to the existing structure. I tried to use the openMVG_main_SfM_Localization method with the 2 new pictures and got an sfm_data_expanded file. My question is can I just call openMVG_main_openMVG2openMVS on this file or do I need to call any other methods on this json file first. Any help is appreciated.

**Comment by pmoulon** [comment]
Yes you need to call `openMVG_main_openMVG2openMVS` on the new produced json file (since it contains the initial scene and the new cameras).

**Comment by vishaal-sridhar** [comment]
I tried this and when I tried to run openMVG_main_openMVG2openMVS on the new json file, I got errors saying it couldn't open the image for each image from the original scene. So the result is, in my output directory, I only have the 2 new images and the scene without any of the original images. And I can't run DensifyPointCloud since it says it's an empty cloud. I tried moving the original images into the folder and it didn't give me that error, but it still has the same problem when I try to run the DensifyPointCloud. Thanks for the help.
